src/macaron/__main__.py

```diff
@@ -244,14 +233,14 @@ def main() -> None:
     )
 
     # Dump the default values.
-    sub_parser.add_parser(name="dump_defaults", description="Dumps the defaults.ini file to the output directory.")
+    sub_parser.add_parser(name="dump-defaults", description="Dumps the defaults.ini file to the output directory.")
 
-    # Verifying a provenance against a policy.
-    verify_parser = sub_parser.add_parser(name="verify")
+    # Verify the Datalog policy.
+    vp_parser = sub_parser.add_parser(name="verify-policy")
```

Comment:
I think we could add the `description` for this command, similar to `dump-defaults` above. 
In additions, the `analzye` command is missing the `description` too. Would you prefer us to address it in another PR?

Reply:
> add the `description`

There is a [ticket](https://github.com/oracle-samples/macaron/issues/211) to address this.